lib/Frontend/Frontend.cpp

```diff
@@ -511,11 +511,11 @@ Optional<CompilerInstance::ModuleBuffers> CompilerInstance::getInputBuffersIfPre
   if (!serialization::isSerializedAST((*inputFileOrErr)->getBuffer()))
     return ModuleBuffers(std::move(*inputFileOrErr));
 
-  return ModuleBuffers(
-    std::move(*inputFileOrErr),
-    openModuleDoc(input).getValueOr(nullptr),
-    openModuleSourceInfo(input).getValueOr(nullptr)
```

Comment:
Boo, this shouldn't formally do a copy. :-( Can we update `LLVM_HAS_RVALUE_REFERENCE_THIS` in Compiler.h instead? Or does MSVC still not have it?

Reply:
Looks like they got re-added. Oh well.

https://github.com/apple/swift-llvm/commit/4ed5b9a3469de38a3cb8120c6f28e1221551a12e